src/index.js

```diff
@@ -0,0 +1 @@
+export * from './subscription';
```

Comment:
just realized that i never came back to give this a better name. anything stand out as a good one?

Reply:
I would think of it as a broker / manager. Essentially the nouns that the software engineering world would tell us to stay away from 😄.

Reply:
what do you think of `subscription-manager.js`?